src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1454,11 +1456,9 @@ bool NetworkOPsImp::beginConsensus (uint256 const& networkClosed)
         return false;
     }
 
-    assert (prevLedger->getHash () == closingInfo.parentHash);
+    assert (prevLedger->info().hash == closingInfo.parentHash);
     assert (closingInfo.parentHash ==
-            m_ledgerMaster.getClosedLedger ()->getHash ());
-
-    prevLedger->setImmutable (app_.config());
```

Comment:
I'm missing why `setImmutable` is no longer needed. I'll look again tomorrow morning.